snow/consensus/snowball/tree_test.go

```diff
@@ -137,14 +137,14 @@ func TestSnowballLastBinary(t *testing.T) {
 		AlphaConfidence: 1,
 		Beta:            2,
 	}
-	tree := NewTree(SnowballFactory, params, zero)
+	tree := NewTree(SnowflakeFactory, params, zero)
 	tree.Add(one)
 
 	// Should do nothing
 	tree.Add(one)
 
-	expected := `SB(PreferenceStrength = 0, SF(Confidence = [0], Finalized = false)) Bits = [0, 255)
-    SB(Preference = 0, PreferenceStrength[0] = 0, PreferenceStrength[1] = 0, SF(Confidence = [0], Finalized = false, SL(Preference = 0))) Bit = 255`
+	expected := `SF(Confidence = [0], Finalized = false) Bits = [0, 255)
```

Comment:
I guess because it's hard to construct an object of a tree that is readable in the test because it contains too many fields, and is of recursive nature.